contracts/GenArt721CoreV3_Engine.sol

```diff
@@ -0,0 +1,2038 @@
+     * has renounced ownership), or conform to IAdminACLV0 interface. Check for
+     * null address first to avoid revert when admin has renounced ownership.
+     */
+    function adminACLAllowed(
+        address _sender,
+        address _contract,
+        bytes4 _selector
+    ) public returns (bool) {
+        return
+            owner() != address(0) &&
+            adminACLContract.allowed(_sender, _contract, _selector);
+    }
+
+    /**
+     * @notice Returns contract owner, as determined by the `superAdmin` of the underlying
+     *         `IAdminACLV0` ACL management contract.
+     * @return address Address of contract owner, as determined by underlying ACL contract
+     * @dev ref: https://docs.openzeppelin.com/contracts/4.x/api/access#Ownable
+     * @dev owner role was called `admin` prior to V3 core contract
+     */
+    function owner()
+        public
+        view
+        override(Ownable, IGenArt721CoreContractV3_Engine)
+        returns (address)
+    {
+        return adminACLContract.superAdmin();
+    }
+
+    /**
+     * @notice Gets token URI for token ID `_tokenId`.
+     * @param _tokenId Token ID to be queried.
+     * @return string URI of token ID `_tokenId`.
+     * @dev token URIs are the concatenation of the project base URI and the
+     * token ID.
+     */
+    function tokenURI(uint256 _tokenId)
+        public
+        view
+        override
+        onlyValidTokenId(_tokenId)
+        returns (string memory)
+    {
+        string memory _projectBaseURI = projects[tokenIdToProjectId(_tokenId)]
+            .projectBaseURI;
+        return string.concat(_projectBaseURI, _tokenId.toString());
+    }
+
+    /**
+     * @dev See {IERC165-supportsInterface}.
+     */
+    function supportsInterface(bytes4 interfaceId)
+        public
+        view
+        virtual
+        override
+        returns (bool)
+    {
+        return
+            interfaceId == type(IManifold).interfaceId ||
+            super.supportsInterface(interfaceId);
+    }
+
+    /**
+     * @notice Forbids new projects from being created
+     * @dev only performs operation and emits event if contract is not already
+     * forbidding new projects.
+     */
+    function _forbidNewProjects() internal {
+        if (!newProjectsForbidden) {
+            newProjectsForbidden = true;
+            emit PlatformUpdated(FIELD_NEW_PROJECTS_FORBIDDEN);
+        }
+    }
+
+    /**
+     * @notice Transfers ownership of the contract to a new account (`newOwner`).
+     * Internal function without access restriction.
+     * @param newOwner New owner.
+     * @dev owner role was called `admin` prior to V3 core contract.
+     * @dev Overrides and wraps OpenZeppelin's _transferOwnership function to
+     * also update adminACLContract for improved introspection.
+     */
+    function _transferOwnership(address newOwner) internal override {
+        Ownable._transferOwnership(newOwner);
```

Comment:
Do we still want the OwnershipTransferred event to fire with the ACL contract address now that owner() is returning the super admin?

Reply:
Resolving, as we've reverted this change based on convo we've had 1:1 offline about the weird implications of returning the `superAdmin`.

Also noted here, for posterity: https://github.com/ArtBlocks/artblocks-contracts/pull/361#issuecomment-1360549182